examples/supported-python-versions-example/control-service-setup/README.md

```diff
@@ -0,0 +1,258 @@
+Supported Python Versions Example PART 1 - Control Service Setup
+
+Overview
+--------
+
+This is the first part of the "Supported Python Versions" example. In this part, we will deploy a Control Service
+instance in a local kubernetes cluster using vdk server, and will update the service to support python3.9 in two
+different vdk configurations, `3.9-pre-release` and `3.9-release`. These would be based on the quickstart-vdk images
+used for the data job deployments.
+
+For this part, we assume that quickstart-vdk is installed and available as described in the [main page](https://github.com/vmware/versatile-data-kit/tree/main/examples/supported-python-versions-example) of the example.
+
+
+Configuration
+-------------
+
+As described above, this example requires the deployment of the VDK Control Service in a kubernetes environment,
+which has certain requirements.
+
+<ins>Prerequisites</ins>:
+
+*   Install [helm](https://helm.sh/docs/intro/install)
+*   Install [docker](https://docs.docker.com/get-docker)
+*   Install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (version 0.11.1 or later)
+
+Once the above prerequisites are fulfilled, install the test Control Service instance by executing
+
+```shell
+vdk server --install
+```
+
+Supported Python Versions
+-------------------------
+
+As mentioned above, for the purpose of this example, we will set up the Control Service to support two different
+versions of quickstart-vdk, 3.9-pre-release and 3.9-release. The difference between the two is that the
+`pre-release` image of quickstart-vdk is updated regularly, while the `release` image has been, at the time of writing,
+last updated around 6 months ago. For both, we will use python3.9-latest as data job base image.
+
+To change the python versions supported by the Control Service, we will set up a _values.yaml_ file with the following
+contents (the ready-to-use file is available in the example [folder](https://github.com/vmware/versatile-data-kit/tree/main/examples/supported-python-versions-example/control-service-setup)):
+
+<details>
+    <summary>values.yaml</summary>
+
+```yaml
+serviceType: ClusterIP
+
+
+# Resources set on all data jobs by default if not overridden per job
+deploymentDefaultDataJobsResources:
+  limits:
+    cpu: 0
+    memory: 0
+  requests:
+    cpu: 0
+    memory: 0
+
+deploymentBuilderResourcesDefault:
+  limits:
+    cpu: 0
+    memory: 0
+  requests:
+    cpu: 0
+    memory: 0
+
+
+## Number of replicas of the Control Plane Deployment
+replicas: 1
```

Comment:
Please omit all non-overriden values. It's abest practice in the local helm values.yaml to set only the overridden values (otherwise you will accidentally override a change in a default value after upgrade that you may not want to override)

Reply:
Cleaned up the file as much as possible